packages/blockchain-wallet-v4-frontend/src/scenes/Dex/Dex.tsx

```diff
@@ -1,94 +1,43 @@
-import React, { useEffect, useState } from 'react'
```

Comment:
Nothing changed in that file in terms of logic. Data related part was just moved to a separate hook.